### ISSUE TITLE
Super, Alt, Ctrl are captured on Linux Server

### ISSUE THREAD

**Issue by siavashs** [debauchee/barrier#936]
**Describe the bug**
I'm using Arch Linux as server and macOS as client, mouse and keyboard control work normally except Super, Alt, Ctrl keys which must trigger Cmd, Alt, Ctrl on macOS.

**To Reproduce**

1. Install Barrier on Server(Linux) and client(macOS)
2. Configure the server & client and start them
3. Move the mouse pointer to client screen(macOS) press Super+Tab buttons
4. The application switch happens on GNOME Shell instead of macOS desktop

**Expected behavior**
Super, Alt, Ctrl must be captured and sent to the client

**Desktop:**
- OS: Linux(server), macOS(client)
- Barrier version 2.3.3

**Comment by claudioap** [comment]
Is your server using wayland? Wayland is unsupported and there's a bounty in #109

**Comment by siavashs** [comment]
Correct, I assumed Wayland is supported since it's the default in many distros now.

**Comment by vii** [comment]
is the bounty for a Wayland client? The issue here is passing through these special keys from the server.

Possible approaches that I can see
- integrate into the Wayland stack (mutter/gnome-shell) and cooperate with them to capture these events
- get into lower level Linux input layer evemu-record /dev/input/event4

This is separate from implementing the client on Wayland

**Comment by metal3d** [comment]
Hi,
Sorry for the message but: it's now 2 years that the problem is reported, any news ? This is the only problem I encounter using Barrier for now… It's a pity.

**Comment by metal3d** [comment]
Note: both server and client on Linux, same problem

**Comment by mitchcurtis** [comment]
Currently encountering an issue where Alt+Tab is captured by the Ubuntu 22.04 server (which uses Wayland by default, apparently), even when the Windows 11 machine has focus. This also results in some strange Alt purgatory, where scrolling in Firefox on Windows results in navigating back and forward through history, changing pages and losing anything that wasn't saved in the process. I found that a workaround is to press Alt once or twice on Windows.

I guess I need to tell it not to use Wayland somehow.
